tests/parser/.gitignore

```diff
@@ -0,0 +1,4 @@
+*
+!Makefile
+!.gitignore
+!*.sby
```

Comment:
Do you have tests to add to this directory already? At least [engines foo] should be working already, no? Also would be good to have parser test cases, just exclude them from CI somehow so that we only manually run them after commenting out the "stages not yet supported" error

Reply:
Yeah this was mainly just in preparation, I wasn't quite sure how to handle the tests due to the fact the additional syntax changes do error out, I'll put some thought into how to go about that and put some preliminary tests in again